lib/src/values/simple_config_object.cc

```diff
@@ -264,6 +264,24 @@ namespace hocon {
         return keys;
     }
 
+    shared_ptr<simple_config_object> simple_config_object::empty() {
+        return empty_instance();
+    }
+
+    shared_ptr<simple_config_object> simple_config_object::empty(shared_origin origin) {
+        if (origin == nullptr) {
+            return empty();
+        } else {
+            // TODO: do this correctly
```

Comment:
If it's wrong, we should probably throw an exception. Or is it currently used in tests?